indico/modules/events/registration/placeholders/registrations.py

```diff
@@ -31,6 +31,20 @@ def render(cls, regform, registration):
         return registration.last_name
 
 
+class PicturePlaceholder(Placeholder):
+    name = 'picture'
+    description = _('Picture of the person')
+
+    @classmethod
+    def render(cls, regform, registration):
+        picture_data = registration.get_personal_data_picture()
+        if not picture_data:
+            return _('NO PICTURE')
+        url = url_for('event_registration.registration_picture', picture_data.locator.file,
+                      _external=True, token=registration.uuid)
```

Comment:
I thought we agreed to use the cid and attach the image to the email? External image references in emails are generally a bad idea.